### DISCUSSION TITLE
Oracle版本驱动问题

### DISCUSSION THREAD

**Discussion by liyang-love** [dotnetcore/FreeSql#1030] [General]
在使用Oracle版本时，我想对旧的项目进行Freesql的迁移，但是由于我是用的Oracle的版本驱动为 4.121.1.0而FreeSql.Provider.Oracle最新版的Oracle驱动绑定死了最小19.3版，导致无法使用，不知道freesql能否像EF或者Dapper那样进行不绑定固定的其他数据库的驱动版本使用。

**Comment by 2881099** [comment]
.UseConnectionFactory(()=>new OracleConnection(…))

或者你去重新编译一个 FreeSql.Provider.Oracle.dll
